content/en/docs/tasks/run-application/configure-pdb.md

```diff
@@ -17,6 +18,8 @@ nodes.
 
 ## {{% heading "prerequisites" %}}
 
+{{< version-check >}}
```

Comment:
I recommend (not essential, but a good idea) that we ask people using older Kubernetes versions to check the documentation for that Kubernetes version.

Reply:
Not sure I understand what we are looking for here. Is there an example I can look at?